docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -815,7 +815,9 @@ WHERE "$file_modified_time" > date_trunc('day', CURRENT_TIMESTAMP);
 (iceberg-optimize-manifests)=
 ##### optimize_manifests
 
-Optimize table manifests to speed up planning.
+Rewrites manifest files to combine them into a single file.
+This command rewrites manifests that are smaller than the `commit.manifest.target-size-bytes`
```

Comment:
I think the property `commit.manifest.target-size-bytes` results in the writer splitting manifest files which exceed the given size threshold rather than acting as a filter for which files we will rewrite.
I would say something like
```
Rewrites manifest files to cluster them by partitioning columns.
This can be used to optimize scan planning when there are many small manifest files or when there are partition filters in read queries but the manifest files are not grouped by partitions.
The iceberg table property `commit.manifest.target-size-bytes` controls 
the maximum size of manifest files produced by this procedure.
```